src/cockpit/bridge.py

```diff
@@ -62,12 +62,12 @@ def export(self, path: str, obj: bus.BaseObject) -> None:
 class Bridge(Router, PackagesListener):
     internal_bus: InternalBus
     packages: Optional[Packages]
-    bridge_rules: List[JsonObject]
+    bridge_configs: List[BridgeConfig]
```

Comment:
The commit message for this part is no longer accurate, since this was never a JsonObject...

Reply:
Updated.